src/components/organisms/BalanceGameSection/BalanceGameSection.tsx

```
@@ -1,240 +1,253 @@
+
+  const gameStages: GameDetail[] =
+    game?.gameDetailResponses ?? gameDefaultDetail;
+  const isGuest = !localStorage.getItem('accessToken');
+
+  const [guestVotedList, setGuestVotedList] = useState<VoteRecord[]>([]);
+    
+  const currentGame: GameDetail = gameStages[currentStage];
+  const { 
+  
+  
+  } = useGuestGameVote(
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)